maintainers/maintainer-list.nix

```diff
@@ -3539,6 +3539,10 @@
     github = "netixx";
     name = "François Espinet";
   };
+  ngerstle = {
+    email = "ngerstle@gmail.com";
+    name = "Nicholas Gerstle";
+  }
```

Comment:
Missing `;` breaks eval, also adding an entry to the maintainer-list should be it's own separate commit.

Reply:
ah, fudge. not a great start :(

Reply:
fixed